test/test.js

```diff
@@ -82,6 +82,49 @@ describe('vdom-streaming-serializer', function(){
 			}
 		});
 	});
+	it('attrib works', function(done){
```

Comment:
Part of the goals of test is being able to understand what is testing. 
Instead of 'attrib works' I would name this test 'streaming attributes work'.
Avoid leaving commented code that is not relevant (Lines 86, 118 and 121). 
Also, avoid having more than one empty line (Lines 102-104, 108-109)